packages/react/src/components/Notification/__tests__/Notification-test.js

```diff
@@ -397,6 +397,8 @@ describe('Callout', () => {
     });
     await userEvent.click(closeButton);
     expect(onActionButtonClick).toHaveBeenCalledTimes(1);
+    expect(onActionButtonClick).toHaveBeenCalledWith(expect.any(Object));
+    expect(onActionButtonClick.mock.calls[0][0].type).toBe('click');
```

Comment:
These three assertions can be reduced to a single assertion:

```js
expect(onActionButtonClick.mock.calls).toEqual([
  [expect.objectContaining({ type: 'click' })],
]);
```

However, I think the `toHaveBeenCalledTimes` check should be kept for consistency with the other tests. With that, the final two assertions can be consolidated to improve accuracy, avoiding `expect.any(Object)`:

```suggestion
    expect(onActionButtonClick).toHaveBeenCalledTimes(1);
    expect(onActionButtonClick.mock.calls).toEqual([
      [expect.objectContaining({ type: 'click' })],
    ]);
```

Reply:
Sure. Updated